lstchain/io/io.py

```diff
@@ -77,6 +77,9 @@
 dl1_params_tel_mon_ped_key = "/dl1/event/telescope/monitoring/pedestal"
 dl1_params_tel_mon_cal_key = "/dl1/event/telescope/monitoring/calibration"
 dl1_params_tel_mon_flat_key = "/dl1/event/telescope/monitoring/flatfield"
+dl1_params_tel_mon_catB_ped_key = "/dl1/event/telescope/monitoring/CatB/pedestal"
```

Comment:
This key is not correct (I assume it is the one that the calibration tool is writing, in that case, I missed it during review, sorry).

It should be `/dl1/monitoring/telescope/...` (not have event!)

Reply:
I see, I will update it